content/blog/balance-le-son.md

```diff
@@ -0,0 +1,50 @@
+---
+title: DevFest 2019 - Balance the sound
+brief: Balance the sound
```

Comment:
ça serait pas mieux "play the guitar" tout simplement ? 🤔  
J'ai peur que ça ne veuille rien dire. Déjà en FR, je comprends pas le "balance le son" 😅